### ISSUE TITLE
API to delete startValues

### ISSUE THREAD

**Issue by arun3688** [OpenModelica/OMSimulator#771]
**Is your feature request related to a problem? Please describe.**
https://github.com/OpenModelica/OMSimulator/issues/748

**Describe the solution you'd like**
The current OMSimulator API `oms_delete()`  will delete models, connectors and its connections and its startValues, But we want to delete StartValues alone , Consider the below example

```
<ssd:Connectors>
    <ssd:Connector name="C3" kind="output">
        <ssc:Real />
    </ssd:Connector>
</ssd:Connectors>
<ssd:ParameterBindings>
    <ssd:ParameterBinding>
        <ssd:ParameterValues>
            <ssv:ParameterSet version="1.0" name="parameters">
                    <ssv:Parameter name="C3">
                        <ssv:Real value="20" />
                    </ssv:Parameter>
                </ssv:Parameters>
            </ssv:ParameterSet>
        </ssd:ParameterValues>
    </ssd:ParameterBinding>
</ssd:ParameterBindings>
```
The current API oms_delete("C3"), will delete Connector and its startValue,  But we want to delete only start value

**Describe alternatives you've considered**
We need to have to an separate API ` oms_deleteStartValues("C3")` or extend the current API `oms_delete` with suffix `start` added to the end some thing like below

`oms_delete('"C3:start")`

What should be a good option ?

**Comment by adeas31** [comment]
Can't we use `oms_setReal`?

**Comment by arun3688** [comment]
`oms_setReal()` sets the value,  but we want to delete it, Do you mean to set the value to `0 `using `oms_setReal("C3", "0") ` ?

**Comment by lochel** [comment]
I am in favor of `oms_delete("C3:start")` because it doesn't require a new API call. I think we have already too many.

**Comment by adeas31** [comment]
> `oms_setReal()` sets the value, but we want to delete it, Do you mean to set the value to `0 `using `oms_setReal("C3", "0") ` ?

Yes. `0` would be anyway assumed default value, rigth?

**Comment by lochel** [comment]
No, it depends on the actual start values in the FMUs.

**Comment by adeas31** [comment]
I mean if there is not actual start value in the FMU.

**Comment by lochel** [comment]
This is implemented in #804
